src/couch_httpd.erl

```diff
@@ -872,7 +872,7 @@ error_info({bad_ctype, Reason}) ->
     {415, <<"bad_content_type">>, Reason};
 error_info(requested_range_not_satisfiable) ->
     {416, <<"requested_range_not_satisfiable">>, <<"Requested range not satisfiable">>};
-error_info({error, illegal_database_name, Name}) ->
+error_info({error, {illegal_database_name, Name}}) ->
```

Comment:
I guess if I provide my own validate_dbname hook with different regexp constraint, this error message will be true no more, right?